lib/handlers/schema.ts

```diff
@@ -15,7 +15,7 @@ export type SupportedPoolInRoute = V2PoolInRoute | V3PoolInRoute | V4PoolInRoute
 
 export type V4PoolInRoute = {
   type: 'v4-pool'
-  address: string
+  poolId: string
```

Comment:
this was discussed with @dannydaniil , and since pool id is never gonna be a ETH address, we decide to return string. follow up to change upstream routing-api response payload as well.

Reply:
there;s type issue, i reverted this change.